scripts/importer.py

```diff
@@ -29,49 +29,162 @@
 import argparse
 
 
+browser_default_input_format = {
+    'chromium': 'netscape',
+    'ie': 'netscape',
+    'firefox': 'netscape',
+    'seamonkey': 'netscape',
+    'palemoon': 'netscape'
+}
+
+
 def main():
     args = get_args()
-    if args.browser in ['chromium', 'firefox', 'ie']:
-        import_netscape_bookmarks(args.bookmarks, args.bookmark_format)
+    bookmark_types = []
+    output_format = None
+    input_format = args.input_format
+    if args.import_search:
+        bookmark_types = ['search']
+        if args.oldconfig:
+            output_format = 'oldsearch'
+        else:
+            output_format = 'search'
+    else:
+        if args.bookmark_output:
+            output_format = 'bookmark'
+        elif args.quickmark_output:
+            output_format = 'quickmark'
+        if args.import_bookmarks:
+            bookmark_types.append('bookmark')
+        if args.import_keywords:
+            bookmark_types.append('keyword')
+    if not bookmark_types:
+        bookmark_types = ['bookmark', 'keyword']
+    if not output_format:
+        output_format = 'quickmark'
+    if not input_format:
+        if not args.browser:
+            print("Must specify either browser or input format")
+            exit(1)
```

Comment:
`exit` is meant for interactive usage in the console, but you can `import sys` and use `sys.exit("Must specify either browser or input format")`.